docs/advanced-guide/handling-file/page.md

```diff
@@ -6,7 +6,7 @@ GoFr simplifies the complexity of working with different file stores by offering
 
 By default, local file-store is initialised and user can access it from the context.
 
-Gofr also supports FTP file-store. The file-store can be initialised with FTP as follows:
+Gofr also supports FTP,SFTP file-store. The file-store can be initialised with FTP as follows:
```

Comment:
I would use this

```suggestion
Gofr also supports FTP/SFTP file-store. The file-store can be initialised as follows:
```

or


```suggestion
Gofr also supports FTP and SFTP file-stores. The file-store can be initialised as follows:
```

Reply:
You resolved, but I didn't see a change

There were two changes I was suggesting:
- FTP/SFTP or FTP and SFTP
- remove the mention of FTP at the end of sentence as the block is now for both

Reply:
@aryanmehrotra please make these changes and update the documentation

Reply:
This was not done.

I commented another PR about FTP to suggest addressing it

https://github.com/gofr-dev/gofr/pull/958#issuecomment-2324018097

Reply:
Missed, will fix in a separate PR.